app/controllers/items_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ItemsController < ApplicationController
+
+  def create
+    @user = User.find(params[:user_id])
+    @item = @user.items.build(item_params)
+    @item.user = current_user
```

Comment:
line 4 just finds the user and sets it to ```@user```. I honestly didn't think I needed line 6 since we are already setting the item's user on line 5